invenio_oaiharvester/tasks/__init__.py

```diff
@@ -96,7 +96,10 @@ def schedule_harvest(output, workflow, directory, name, records):
         print_total_records(total)
     elif output == 'workflow':
         workflow_name = get_workflow_name(workflow, name)
+        total = 0
         for record in records:
             start_delayed(workflow_name, [record.raw])
+            total += 1
+        print_total_records(total)
```

Comment:
It's a generator. I could do `len(list(records))`, but it would defeat the purpose of the generator.